src/core/manager.h

```diff
@@ -254,6 +254,8 @@ struct Manager {
 
         bool taint_usr:1;
 
+        bool ready_sent:1;
```

Comment:
hmm, i figure we should serialize this bool during daemon reload

Reply:
(admittedly sending out READY=1 twice isn't too bad, but still)

Reply:
Updated.